backend/src/BuiltinExecution/Libs/Char.fs

```diff
@@ -45,78 +45,71 @@ let fns : List<BuiltInFn> =
       deprecated = NotDeprecated }
 
 
-    { name = fn "isDigit" 0
+    { name = fn "isASCII" 0
```

Comment:
As a followup, could isASCII be implemented in Dark?

Reply:
I couldn't find a good way to say that it should be between NUL and DEL or 0 and 127.
I think we'll need a new Builtin function that returns the ASCII value of a character.
Which means adding a new builtin function to remove one. Thoughts?
the functions is something like :
```
(if c.Length = 1 then System.Convert.ToInt64 c[0] else -1) |> DInt |> Ply
```
And the dark version of isAscii  will be 
```
      let isASCII (c: Char) : Bool =
        let ascii = Builtin.Char.asciiOfChar c
        (ascii >= 0) && (ascii <= 127)
```

Reply:
Yeah, that's a good function to add.

I'd call it Char.toAsciiCode and it should return an option as characters might not be ascii at all.